src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -177,6 +177,10 @@ class DefaultStorageKeyAllocator
     static StorageKeyName BindingTable() { return StorageKeyName::FromConst("g/bt"); }
     static StorageKeyName BindingTableEntry(uint8_t index) { return StorageKeyName::Formatted("g/bt/%x", index); }
 
+    // Client Monitoring
+
+    static StorageKeyName ClientMonitoringTableEntry(chip::FabricIndex fabric) { return StorageKeyName::FromConst("f/%x/cm"); }
```

Comment:
This is not going to do the right thing.  You need to use `Formatted` here, not `FromConst`, I would think.

If the tests did not catch this, we need to fix the tests.

Reply:
I think the tests didn't catch it because the test cases only test for 1 fabric so there can't be any key conflicts. I will add a test to write / read two different fabrics.

Reply:
done